packages/react-core/src/helpers/util.ts

```diff
@@ -107,14 +107,13 @@ export function sideElementIsOutOfView(container: HTMLElement, element: HTMLElem
  *    const result = fillTemplate(templateString, templateVars);
  *    // "My name is Jon Dough"
  *
- * @param {object} templateString  The string passed by the consumer
+ * @param {string} templateString  The string passed by the consumer
  * @param {object} templateVars The variables passed to the string
  *
  * @returns {string} The template string literal result
  */
 export function fillTemplate(templateString: string, templateVars: any) {
-  const func = new Function(...Object.keys(templateVars), `return \`${templateString}\`;`);
-  return func(...Object.values(templateVars));
+  return templateString.replace(/\${(.*?)}/g, (_, match) => templateVars[match] || '');
```

Comment:
would be good to have some tests for this, and verify before and after is the same

Reply:
We have this test: https://github.com/patternfly/patternfly-react/blob/v4/packages/react-core/src/helpers/__tests__/util.test.js#L217

Also we have snapshots for our 3 components that use this helper that remain unchanged. What other tests would you like to see?

Reply:
ah good, all set then